src/core/components/NavBar/index.js

```diff
@@ -1,38 +0,0 @@
-import React from 'react';
```

Comment:
There's an issue open for it this PR marks as fixing... I caught it awhile back. I'm just doing it in this PR because that component's tests used our old `shallowRender` so I deleted it here rather than fixing its tests only to delete it later 😄